package.json

```diff
@@ -1,23 +1,32 @@
 {
-  "name": "ts-template",
+  "name": "@ubiquity/rpc-handler",
```

Comment:
Should be `@ubiquitydao/rpc-handler`

Reply:
but the org it's named **ubiquity** here on github

Reply:
> but the org it's named **ubiquity** here on github

We just created https://www.npmjs.com/org/ubiquitydao and this will be deployed to npm there!

Reply:
Sadly @ubiquity already exists but no one seems to know who owns it